acceptancetests/jujupy/k8s_provider/base.py

```diff
@@ -211,13 +212,14 @@ def patch_configmap(self, namespace, cm_name, key, value):
         cm['data'] = data
         self.kubectl_apply(json.dumps(cm))
 
-    def sh(self, *args):
-        args = [str(arg) for arg in args]
+    def sh(self, *args, shell=False):
+        args = [quote(str(arg)) for arg in args]
```

Comment:
We only want to quote the args if shell is True, right?

Reply:
ah, right. ty!